ui/apps/platform/cypress/integration/integrations/integrations.helpers.js

```diff
@@ -120,6 +122,50 @@ function getIntegrationEndpointAddress(integrationSource, integrationType, integ
 }
 
 // Please forgive such an abstract definition.
+
+const routeMatcherMapForIntegrationsTab = {
+    imageIntegrations: {
+        [getIntegrationsEndpointAlias('imageIntegrations')]: {
```

Comment:
Thought for the future, not for this PR.

Would we lose any re-usability if these were simplified to constants instead of lookup functions?
```diff
- getIntegrationsEndpointAlias('imageIntegrations')
+ imageIntegrationsEndpointAlias
```
```diff
- getIntegrationsEndpointAddressForGET('imageIntegrations')
+ imageIntegrationsEndpointAddressForGET
```